style.css

```diff
@@ -0,0 +1,488 @@
+/* ElementX font */
+@import url('https://fonts.googleapis.com/css2?family=Orbitron:wght@700&display=swap');
+
+/* Font for index */
+@import url('https://fonts.googleapis.com/css2?family=Hind&display=swap');
+
+/* Font for Content */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;700&display=swap');
+
+:root {
+    --gray: #6B7280;
+    --dark-gray: #374151;
+    --dark-red: #DC2626;
+    --light-red: #FCA5A5;
+    --light-gray: #D1D5DB;
+    --yellow: #F59E0B;
+    --light-yellow: #FDE68A;
+    --blue: #2563EB;
+    --dark-blue: #1E40AF;
+    --light-blue: #93C5FD;
```

Comment:
In future you can surely refactor this large file into smaller component specific css files and import it in the styles.css